Bugzilla 562. When the MX31 is suspended it was resuming on ethernet
activity due to a CPLD interrupt.  This patch fixes the problem.
========================================================================
diff -uNpr linux-2.6.19.2.orig/arch/arm/mach-mx3/mxc_pm.c linux-2.6.19.2/arch/arm/mach-mx3/mxc_pm.c
--- linux-2.6.19.2.orig/arch/arm/mach-mx3/mxc_pm.c	2007-05-09 18:10:33.000000000 -0600
+++ linux-2.6.19.2/arch/arm/mach-mx3/mxc_pm.c	2007-05-09 18:07:42.000000000 -0600
@@ -280,6 +280,7 @@ void mxc_pm_lowpower(int mode)
 		/* State Retention mode */
 		lpm = 2;
 		__raw_writel(INT_GPT, AVIC_INTDISNUM);
+		__raw_writel(INT_GPIO1, AVIC_INTDISNUM);
 
 		/* work-around for SR mode after camera related test */
 		__raw_writel(0x51, IPU_CONF);
@@ -316,6 +317,7 @@ void mxc_pm_lowpower(int mode)
 	__raw_writel(ipu_conf, IPU_CONF);
 
 	__raw_writel(INT_GPT, AVIC_INTENNUM);
+	__raw_writel(INT_GPIO1, AVIC_INTENNUM);
 
 	local_irq_enable();
 }
